lego/apps/pinned/models.py

```diff
@@ -0,0 +1,19 @@
+from django.db import models
+
+from lego.apps.articles.models import Article
+from lego.apps.events.models import Event
+from lego.apps.users.models import AbakusGroup
+from lego.utils.models import BasisModel
+
+
+class Pinned(BasisModel):
+    pinned_from = models.DateField()
+    pinned_to = models.DateField()
+
+    event = models.ForeignKey(
+        Event, related_name='pins', on_delete=models.SET_NULL, null=True, blank=True
```

Comment:
Maybe cascade these, so we don't end up with `pinned` values without an event or article?